### PR TITLE
updated hello.md

### PR DESCRIPTION
**Pull request by eugenetaan**

### CODE REVIEW

**Comment by rldefa** [comment]
merge conflict

**Comment by eugenetaan** [comment]
Resolved Conflicts

File: hello.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # hello
+<<<<<<< HEAD
```
Comment (rldefa) [inline hello.md]: Remove all these that were added in by Git.

File: hello.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 My name is Cow.
 
 I like to play with RHesidents.
 
 I think I am a cow.
+=======
```
Comment (rldefa) [inline hello.md]: same for this line

**Review by rldefa — CHANGES_REQUESTED** [review]

**Comment by rldefa** [comment]
Also, you are supposed to keep the one you wrote yourself not the one in submit.md

**Comment by eugenetaan** [comment]
Edited to my own one and removed git changes

**Comment by rldefa** [comment]
lgtm

**Comment by rldefa** [comment]
Except for the fact u updated hello.md on github please don't make that a habit. Apart from that alls good

**Comment by rldefa** [comment]
Close one eye for now but DONT do everything on github.

**Pull request closed without merging**